### ISSUE TITLE
toJson() doesn't remove "kw$" prefix

### ISSUE THREAD

**Issue by kono0514** [comigor/artemis#203]
**Bug description**
```r'kw$operator'``` should be ```'operator'``` in this case:
```dart
// graphql_api.graphql.g.dart
Map<String, dynamic> _$ArticleTitleWhereConditionsToJson(
        ArticleTitleWhereConditions instance) =>
    <String, dynamic>{
      r'kw$operator': _$SQLOperatorEnumMap[instance.kw$operator],
      'value': instance.value,
    };
```

**Specs**
Artemis version: ^6.8.2-beta.1
<details>
<summary>build.yaml:</summary>
  
```yaml
targets:
  $default:
    sources:
      - lib/**
      - graphql/**
      - schema.graphql
    builders:
      artemis:
        options:
          schema_mapping:
            - schema: schema.graphql
              queries_glob: graphql/articles.graphql
              output: lib/graphql_api.graphql.dart
```
</details>
<details>
<summary>Artemis output:</summary>
  
```bash
[ +163 ms] executing: [C:\flutter\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[  +78 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +3 ms] 2ae34518b87dd891355ed6c6ea8cb68c4d52bb9d
[   +2 ms] executing: [C:\flutter\flutter/] git tag --contains HEAD
[ +251 ms] Exit code 0 from: git tag --contains HEAD
[   +1 ms] 1.20.1
[  +11 ms] executing: [C:\flutter\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +67 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[   +1 ms] origin/stable
[        ] executing: [C:\flutter\flutter/] git ls-remote --get-url origin
[  +46 ms] Exit code 0 from: git ls-remote --get-url origin
[   +7 ms] https://github.com/flutter/flutter.git
[ +110 ms] executing: [C:\flutter\flutter/] git rev-parse --abbrev-ref HEAD
[  +60 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[   +2 ms] stable
[ +139 ms] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[  +85 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[   +3 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[  +30 ms] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[  +15 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[  +42 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[   +6 ms] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[  +77 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[        ] Artifact Instance of 'AndroidMavenArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +1 ms] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[   +1 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[  +13 ms] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[  +15 ms] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[   +1 ms] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[  +13 ms] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[   +1 ms] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[  +14 ms] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[  +60 ms] Using C:\flutter\flutter\.pub-cache for the pub cache.
[   +7 ms] executing: C:\flutter\flutter\bin\cache\dart-sdk\bin\pub.bat run build_runner build --verbose
Precompiling executable...
Precompiled build_runner:build_runner.
[INFO] Generating build script...
[INFO] Generating build script completed, took 485ms

[WARNING] build.fallback:
The package `flutter_artemis_test` does not include some required sources in any of its targets (see their build.yaml file).
The missing sources are:
  - $package$
[INFO] BuildDefinition:Initializing inputs
[INFO] BuildDefinition:Reading cached asset graph...
[INFO] BuildDefinition:Reading cached asset graph completed, took 147ms

[INFO] BuildDefinition:Checking for updates since last build...
[INFO] BuildDefinition:Checking for updates since last build completed, took 932ms

[WARNING] BuildDefinition:
Invalidating asset graph due to build script update!
[INFO] BuildDefinition:Cleaning up outputs from previous builds....
[INFO] BuildDefinition:Cleaning up outputs from previous builds. completed, took 11ms

[INFO] Generating build script...
[INFO] Generating build script completed, took 139ms

[INFO] Creating build script snapshot......
[INFO] Creating build script snapshot... completed, took 28.2s

[WARNING] build.fallback:
The package `flutter_artemis_test` does not include some required sources in any of its targets (see their build.yaml file).
The missing sources are:
  - $package$
[INFO] BuildDefinition:Initializing inputs
[INFO] BuildDefinition:Building new asset graph...
[INFO] BuildDefinition:Building new asset graph completed, took 1.2s

[INFO] BuildDefinition:Checking for unexpected pre-existing outputs....
[INFO] BuildDefinition:Checking for unexpected pre-existing outputs. completed, took 2ms

[INFO] Build:Running build...
[FINE] artemis:artemis on lib/$lib$:-> Input class
[FINE] artemis:artemis on lib/$lib$:┌ [][ArticleTitleWhereConditions]
[FINE] artemis:artemis on lib/$lib$:|   [][ArticleTitleWhereConditions][ClassName(name:r'SQLOperator') ClassPropertyName(name:r'operator')] -> SQLOperator
[FINE] artemis:artemis on lib/$lib$:|   [][ArticleTitleWhereConditions][ClassName(name:r'String') ClassPropertyName(name:r'value')] -> String
[FINE] artemis:artemis on lib/$lib$:└ [][ArticleTitleWhereConditions]
[FINE] artemis:artemis on lib/$lib$:<- Generated input class ArticleTitleWhereConditions.
[FINE] artemis:artemis on lib/$lib$:-> Enum
[FINE] artemis:artemis on lib/$lib$:<- Generated enum SQLOperator.
[FINE] artemis:artemis on lib/$lib$:-> Class
[FINE] artemis:artemis on lib/$lib$:┌ [TypeName(name:r'SearchArticles'), TypeName(name:r'Query')][Query][null null] ()
[FINE] artemis:artemis on lib/$lib$:|   [TypeName(name:r'SearchArticles'), TypeName(name:r'Query')][Query][ClassName(name:r'Article') ClassPropertyName(name:r'articles')] -> List<SearchArticles$Query$Article>
[FINE] artemis:artemis on lib/$lib$:|   -> Class
[FINE] artemis:artemis on lib/$lib$:|   ┌ [TypeName(name:r'SearchArticles'), TypeName(name:r'Query'), ClassName(name:r'Article')][Article][null null] ()
[FINE] artemis:artemis on lib/$lib$:|   |   [TypeName(name:r'SearchArticles'), TypeName(name:r'Query'), ClassName(name:r'Article')][Article][ClassName(name:r'ID') ClassPropertyName(name:r'id')] -> String
[FINE] artemis:artemis on lib/$lib$:|   |   [TypeName(name:r'SearchArticles'), TypeName(name:r'Query'), ClassName(name:r'Article')][Article][ClassName(name:r'String') ClassPropertyName(name:r'title')] -> String
[FINE] artemis:artemis on lib/$lib$:|   └ [TypeName(name:r'SearchArticles'), TypeName(name:r'Query'), ClassName(name:r'Article')][Article][null null] ()
[FINE] artemis:artemis on lib/$lib$:|   <- Generated class SearchArticles$Query$Article.
[FINE] artemis:artemis on lib/$lib$:└ [TypeName(name:r'SearchArticles'), TypeName(name:r'Query')][Query][null null] ()
[FINE] artemis:artemis on lib/$lib$:<- Generated class SearchArticles$Query.
[INFO] Heartbeat:1.3s elapsed, 1/3 actions completed.
[INFO] Heartbeat:2.5s elapsed, 1/3 actions completed.
[INFO] Heartbeat:4.2s elapsed, 1/3 actions completed.
[INFO] Heartbeat:6.1s elapsed, 1/3 actions completed.
[FINE] json_serializable:json_serializable on lib/graphql_api.graphql.dart:Running JsonSerializableGenerator - 1 of 2
[FINE] json_serializable:json_serializable on lib/graphql_api.graphql.dart:Running JsonLiteralGenerator - 2 of 2
[FINE] json_serializable:json_serializable on lib/main.dart:Running JsonSerializableGenerator - 1 of 2
[FINE] json_serializable:json_serializable on lib/main.dart:Running JsonLiteralGenerator - 2 of 2
[INFO] Build:Running build completed, took 12.1s

[INFO] Build:Caching finalized dependency graph...
[INFO] Build:Caching finalized dependency graph completed, took 53ms

[INFO] Build:Succeeded after 12.2s with 3 outputs (6 actions)

[+69343 ms] "flutter run" took 69,801ms.
[ +258 ms] ensureAnalyticsSent: 252ms
[   +4 ms] Running shutdown hooks
[   +9 ms] Shutdown hooks complete
[   +2 ms] exiting with code 0
```
</details>
<details>
<summary>GraphQL schema:</summary>

```graphql
type Query {
  articles(titleWhere: ArticleTitleWhereConditions): [Article!]
}

input ArticleTitleWhereConditions {
  operator: SQLOperator
  value: String
}

type Article {
  id: ID!
  title: String!
}

enum SQLOperator {
  EQ
  IN
}
```
</details>
<details>
<summary>GraphQL query:</summary>

```graphql
query SearchArticles($titleWhere: ArticleTitleWhereConditions) {
    articles(titleWhere: $titleWhere) {
        id
        title
    }
}
```
</details>

**Comment by kono0514** [comment]
> Applied proper fix. Use that branch until next relesae

Thank you!

**Comment by vasilich6107** [comment]
Fixed in `6.9.2-beta.1`
